### DISCUSSION TITLE
Can't pull from Git in Arduino IDE

### DISCUSSION THREAD

**Discussion by prayingmantra** [espressif/arduino-esp32#10984] [Announcements]
### Board

ESP32 WROOM 32

### Device Description

Devkit C

### Hardware Configuration

N/A

### Version

v3.0.3

### IDE Name

Arduino IDE

### Operating System

Windows 10

### Flash frequency

40 Mhz

### PSRAM enabled

yes

### Upload speed

115200

### Description

Error thrown when trying to update boards. 
Failed to install platform: 'esp32:esp32:3.0.2'.
Error: 2 UNKNOWN: read tcp 192.168.178.187:63599->185.199.109.133:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

### Sketch

```cpp
N/A
```

### Debug Message

```plain
N/A
```

### Other Steps to Reproduce

 N/A

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by VojtechBartoska** [comment]
Hello, there will be something wrongly setup at your environment.

Take a look on this guide if you have proper link for packages.

https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html
